src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.container {
```

Comment:
it looks like separated block as well, pay more attention while creating files on next time